src/components/additem/AddProduct.jsx

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useRef } from "react";
+import styles from "./AddProduct.module.css";
+
+const AddProduct = ({
+  productImage,
+  productTitle,
+  productDescription,
+  productPrice,
+  productTags,
```

Comment:
P2:
속성값을 모두 받는 것보다 아래처럼 disabled 값을 받아서 전달만 해주시면 좋겟습니다.
해당 컴포넌트에서 이러한 속성을 모두 알 필요가 없으니까요~

```suggestion
  isDisabled
```